src/Microsoft.FileFormats/TStruct.cs

```diff
@@ -145,20 +145,33 @@ public static LayoutManager AddTStructTypes(this LayoutManager layouts, IEnumera
         /// <returns></returns>
         public static LayoutManager AddReflectionTypes(this LayoutManager layouts, IEnumerable<string> enabledDefines, Type requiredBaseType)
         {
-            layouts.AddLayoutProvider((type, layoutManager) => GetTStructLayout(type, layoutManager, enabledDefines, requiredBaseType));
+            layouts.AddLayoutProvider((type, layoutManager) =>
```

Comment:
Can this one call the other AddReflectionTypes with `typeFilter: (type) => requiredBaseType.GetTypeInfo().IsAssignableFrom(type)`

Reply:
Will do.